pkg/config/config_test.go

```diff
@@ -1049,3 +1049,83 @@ func (s *MySuite) TestValidatorConfigCheck(c *C) {
 		c.Check(err, ErrorMatches, "only 3 inputs \\[in0 in1 in2\\] should be provided to test_project_exists")
 	}
 }
+
+func (s *MySuite) TestCheckBackends(c *C) {
+	// Helper to create blueprient with backend blocks only (first one is defaults)
+	// and run checkBackends.
+	check := func(d TerraformBackend, gb ...TerraformBackend) error {
+		gs := []DeploymentGroup{}
+		for _, b := range gb {
+			gs = append(gs, DeploymentGroup{TerraformBackend: b})
+		}
+		bp := Blueprint{
+			TerraformBackendDefaults: d,
+			DeploymentGroups:         gs,
+		}
+		return checkBackends(bp)
+	}
+	dummy := TerraformBackend{}
+
+	{ // Absent
+		c.Check(checkBackends(Blueprint{}), IsNil)
+	}
+
+	{ // Dummies
```

Comment:
should probably clarify that empty backends should pass

Reply:
Added OK/FAIL mark to each case, modified some comments.